news/2024-05-15-numpy-2-migration.md

```diff
@@ -0,0 +1,9 @@
+# NumPy 2 Migration
+
+In preparation for the NumPy 2 release, we have begun a migration of all
+conda-forge packages that depend on `numpy` during the build (so in
+`requirements/host`). The migration status is tracked
+[on our status page](https://conda-forge.org/status/migration/numpy2).
```

Comment:
> The problem is that the migration details URLs are generated dynamically on the JS client, but a static request gives a 404, so they will always fail the checker.

That's fair. Would something like this work?

```suggestion
[on our status page](./status/migration/numpy2).
```

Reply:
cc @jaimergp

Reply:
```suggestion
[on our status page](/status/migration/numpy2).
```

This is better. You might still need the lychee exclude though.